include/Core/TransparentImageBase64.hpp

```diff
@@ -0,0 +1,9 @@
+#ifndef CORE_TRANSPARENT_IMAGE_BASE64_HPP
+#define CORE_TRANSPARENT_IMAGE_BASE64_HPP
+
+// A transparent image base64 string.
+// Since we want to hardcode the image, we have such a long string here.
+// The original image should find in here: https://i.imgur.com/zS4sPCN.png
+static constexpr const char *TRANSPARENT_IMAGE = "iVBORw0KGgoAAAANSUhEUgAAAQAAAAEABAMAAACuXLVVAAAAIGNIUk0AAHomAACAhAAA+gAAAIDoAAB1MAAA6mAAADqYAAAXcJy6UTwAAAAhUExURf8A3P8A3uEAwh4AGgAAAP8A3+IAwx0AGcoArjUALv///xhkxvUAAAABYktHRApo0PRWAAAAB3RJTUUH5wwZEiAsl4pL0QAAAQlJREFUeNrt2rENAjEMBVCvcGzArRA2IBIbsAESA9CwACuwLrVDnfiK5/YX9xRdY+tH5DldeprreW7eAwAAAAAAAAAAAAAAAACgHLDlub9yfnvMzXvseZ7flub9mZu3aGs/OOZ79LVPPubbCJj90415AAAAAAAAAAAAAAAAABwOsH45LV/Pyw8U5SeaWPvTHfBKBgAAAAAAAAAAAAAAAFC+nJav5+UHir72yfUHAAAAAAAAAAAAAAAAAPQH9Af0BwAAAAAAAAAAAAAAAAD0B/QH9AcAAAAAAAAAAAAAAAAA9Af0B/QHAAAAAAAAAAAAAAAAAPQH9Af0BwAAAAAAAAAAAAAAAAAA/gA/YLVKK0nwuR8AAAAldEVYdGRhdGU6Y3JlYXRlADIwMjMtMTItMjVUMTg6MzI6MzcrMDA6MDCbYQaeAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDIzLTEyLTI1VDE4OjMyOjM3KzAwOjAw6jy+IgAAACh0RVh0ZGF0ZTp0aW1lc3RhbXAAMjAyMy0xMi0yNVQxODozMjo0NCswMDowMIYEjHkAAAAASUVORK5CYII=";
+
+#endif // CORE_TRANSPARENT_IMAGE_BASE64_HPP
```

Comment:
Blank line at EOF as mentioned in [Code Styles and Guidelines](https://hackmd.io/bOzigaM9R5Sgsl0nqMtRQg).

Reply:
Fix on 9958cda